horreum-web/src/domain/actions/reducers.ts

```diff
@@ -85,6 +86,23 @@ export const testEventTypes = [
   }
 }`,
   ],
+  [
+      DATASET_LABELS_COMPUTED,
+      `{
+    "id": 123,
+    "data": ...,
+    "start": "2022-08-29T12:28:14+02:00",
+    "stop": "2022-08-29T12:28:14+02:00",
+    "owner": "dev-team",
+    "access": 0,
+    "schema": {
+      "1": "urn:my-schema:1.0"
+    },
+    "testid": 10,
+    "testname": "My test",
+    "description": null,
+  }`,
+    ],
```

Comment:
The object should match the `Dataset` object structure, this is actually just an example with fake data